samples/cellular/modem_shell/src/gpio_count/gpio_count.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef MOSH_GPIO_COUNT_H
+#define MOSH_GPIO_COUNT_H
+
+#include <zephyr/types.h>
+
+int gpio_count_enable(uint8_t pin);
+int gpio_count_disable(void);
+uint32_t gpio_count_get(void);
+void gpio_count_reset(void);
```

Comment:
I probably would have left all these functions into `gpio_count_shell.c` as we have done in the small command modules. But I'll leave it to you to decide.

Reply:
I did consider whether to have separate file for the GPIO implementation or not. It is there now and moving the implementation to `gpio_count_shell.c` would cause some work and I don't see any clear reason for doing it. So, I'm keeping it as it is.